sycamore/functions/chunker.py

```diff
@@ -9,7 +9,25 @@ def chunk(self, tokens: list[Any]) -> list[Any]:
 
 
 class TokenOverlapChunker(Chunker):
-    def __init__(self, chunk_token_count=1000, chunk_overlap_token_count=100) -> None:
+    """
+    TokenOverlapChunker is a class for chunking tokens into smaller segments while allowing for token overlap.
```

Comment:
This is really for chunking text rather than tokens. In fact token is a bit of a misnomer here, since we aren't using an actual tokenizer -- just words/characters.

Reply:
Updated